src/App.js

```diff
@@ -8,8 +8,8 @@ import Login from "./pages/Login";
 import Signup from './pages/Signup';
 
 function App(props) {
-  const [authTokens, setAuthTokens] = useState();
-  
+  const [authTokens, setAuthTokens] = useState(localStorage.getItem('authTokens') || '');
+
   const setTokens = (data) => {
     localStorage.setItem("tokens", JSON.stringify(data));
```

Comment:
Not the repo owner, but shouldn't the localstorage variable "tokens" be the same "authTokens" on line 11?

Reply:
@vlod: Thanks for noticing! Fixed it.